test/test_cuda.py

```diff
@@ -1728,7 +1728,6 @@ def test_multinomial_invalid_probs_cuda(self):
         self._spawn_method(test_method, torch.Tensor([1, inf, 1]))
         self._spawn_method(test_method, torch.Tensor([1, -inf, 1]))
         self._spawn_method(test_method, torch.Tensor([1, 1, nan]))
-        self._spawn_method(test_method, torch.Tensor([0, 1, 0]))
```

Comment:
Not sure why this was there as zero should be a valid value. :thinking: 
https://github.com/pytorch/pytorch/blob/db2fd452c8a8591b02a57afd694bb0e94cf69357/aten/src/ATen/native/cuda/MultinomialKernel.cu#L189-L192

Reply:
Yeah, it's ok to remove this test, it looks like it should be valid.